README.md

```diff
@@ -624,6 +625,24 @@ This will transform `<disallowed>content</disallowed>` to `&lt;disallowed&gt;con
 
 Valid values are: `'discard'` (default), `'escape'` (escape the tag) and `'recursiveEscape'` (to escape the tag and all its content).
 
+### Ignore style attribute contents
+
+Instead of discarding faulty style attributes, you can allow them by disabling the parsing of style attributes:
+
+```js
+parseStyleAttributes: false
+```
+
+This will transform `<div style="invalid-prop: non-existing-value">content</div>` to `<div style="invalid-prop: non-existing-value">content</div>` instead of stripping it: `<div>content</div>`
+
+By default the parseStyleAttributes option is true.
+
+When you disable the parsing of the style attribute, the `allowedStyles` option is automatically ignored, and all styles will be allowed.
+
+Note that by default this library also strips style tags when you run this package in the browser. Because of a postcss dependency. https://github.com/apostrophecms/sanitize-html/issues/547
```

Comment:
I don't think this is really an accurate description of the problem, but I get that it is rather difficult to pin down and maybe doesn't belong in the README as it could get fixed upstream. So I suggest we lose this paragraph, but see my notes on the next one.